tests/update/tests.py

```diff
@@ -249,6 +249,13 @@ def test_update_ordered_by_m2m_annotation(self):
         Bar.objects.annotate(abs_id=Abs("m2m_foo")).order_by("abs_id").update(x=3)
         self.assertEqual(Bar.objects.get().x, 3)
 
+    def test_update_ordered_by_m2m_annotation_desc(self):
+        foo = Foo.objects.create(target="test")
+        Bar.objects.create(foo=foo)
+
+        Bar.objects.annotate(abs_id=Abs("m2m_foo")).order_by("-abs_id").update(x=4)
+        self.assertEqual(Bar.objects.get().x, 4)
```

Comment:
This test should be skipped on MySQL/MariaDB, add it to the:

https://github.com/django/django/blob/d2b688b966f5d30414899549412d370e1317ddb8/django/db/backends/mysql/features.py#L106-L111